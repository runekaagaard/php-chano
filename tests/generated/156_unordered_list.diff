001- <li>item 1
001+ Fatal error: Call to undefined method DtlIter::unordered_list() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/django-generated/156_unordered_list.php on line 12
002- 	<ul>
003- 		<li>item 1.1</li>
004- 		<li>item1.2</li>
005- 	</ul>
006- 	</li>
007- 	<li>item 2</li>