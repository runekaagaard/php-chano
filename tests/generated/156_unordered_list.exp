<li>item 1
	<ul>
		<li>item 1.1</li>
		<li>item1.2</li>
	</ul>
	</li>
	<li>item 2</li>