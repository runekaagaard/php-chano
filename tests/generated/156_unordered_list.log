
---- EXPECTED OUTPUT
<li>item 1
	<ul>
		<li>item 1.1</li>
		<li>item1.2</li>
	</ul>
	</li>
	<li>item 2</li>
---- ACTUAL OUTPUT
Fatal error: Call to undefined method DtlIter::unordered_list() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/generated/156_unordered_list.php on line 12
---- FAILED
