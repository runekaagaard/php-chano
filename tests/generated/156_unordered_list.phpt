--TEST--
A generated testfile for the "unordered_list" filter.
--FILE--
<?php
include dirname(__FILE__) . '/../../DtlIter.php';
$items = new DtlIter(array(array('input' => array (
  0 => 'item 1',
  1 => 
  array (
    0 => 'item 1.1',
    1 => 'item1.2',
  ),
  2 => 'item 2',
))));
foreach ($items as $i) echo $i->input->unordered_list();
--EXPECT--
	<li>item 1
	<ul>
		<li>item 1.1</li>
		<li>item1.2</li>
	</ul>
	</li>
	<li>item 2</li>