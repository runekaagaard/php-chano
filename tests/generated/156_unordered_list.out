Fatal error: Call to undefined method DtlIter::unordered_list() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/django-generated/156_unordered_list.php on line 12